cmd/flow-aggregator/options.go

```diff
@@ -57,6 +57,8 @@ type Options struct {
 	flowAggregatorAddress string
 	// Format for record sent to the configured flow collector
 	format string
+	// Whether sourcePodLabels and destinationPodLabels will be included
```

Comment:
"includePodLabels indicates whether source and destination Pod labels are included or not"

Reply:
done